kpi/maintenance_tasks.py

```diff
@@ -0,0 +1,39 @@
+import constance
+from datetime import timedelta
+
+from django.db.models import Max
+from django.utils import timezone
+
+from kpi.models import AssetSnapshot
+
+
+default_retention_period = 90
+
+def remove_old_assetsnapshots():
+    days = (
+        constance.config.ASSET_SNAPSHOT_DAYS_RETENTION
+        or default_retention_period
```

Comment:
I believe the way constance works is that it uses a default already, 30 in this case. Maybe double check that. I suspect the 90 here will never be used.